server/index.js

```diff
@@ -1,18 +1,48 @@
 const express = require('express');
-const port = 4004;
-const app = express();
 const db = require('../database/database.js');
 const seeder = require('../database/seeding.js');
 const bodyParser = require('body-parser');
+const query = require('../database/query.js');
+const cors = require('cors');
+const morgan = require('morgan');
+require("dotenv").config();
 
+const app = express();
 app.use(express.static(__dirname + '/../client/dist'));
-app.use(bodyParser.urlencoded({extended: false}));
+app.use(bodyParser.urlencoded({extended: true}));
 app.use(bodyParser.json());
+// app.use(cors());
+app.use(morgan('tiny'));
+
+
+
+//get one item's description
+app.get('/description/:productId', (req, res, next) => {
+  let productId = req.params.productId;
+  query.getDescriptionForOneProduct(productId, (err, description) => {
+    if (description.length === 0) {
+      res.sendStatus(404);
+      next();
+    } else {
+      res.status(200).send(description);
+      next();
+    }
+  });
+});
 
-app.get('/description/1000', (req, res) => {
-  var data = db.findOne({productId: 1000});
-  res.send(data);
+//get multiple item descriptions
+app.post('/description/multiple', (req, res, next) => {
```

Comment:
Is there a reason for using a post request here? The function looks like it is querying and reading data, as opposed to inserting data.

Reply:
I think you're right that a "get" request makes more sense here. I'm going to go ahead and make the change.